bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -1069,7 +1129,7 @@ public void sendAnnouncement(Device device, String text, @Nullable String title,
 
     public void textToSpeech(Device device, String text, int ttsVolume, int standardVolume)
             throws IOException, URISyntaxException {
-        Map<String, Object> parameters = new Hashtable<>();
+        Map<String, Object> parameters = new Hashtable<String, Object>();
```

Comment:
Why did you step away from the diamond-operator? Also please do not to use Hashtable its old and kind of deprecated, from the manual:

> Unlike the new collection implementations, Hashtable is synchronized. If a thread-safe implementation is not needed, it is recommended to use HashMap in place of Hashtable. If a thread-safe highly-concurrent implementation is desired, then it is recommended to use ConcurrentHashMap in place of Hashtable.

Reply:
Done